src/p_mobj.c

```diff
@@ -1209,10 +1209,11 @@ void P_SpawnMapThing (mapthing_t* mthing)
     return;
 
   // killough 11/98: simplify
-  if (gameskill == sk_baby || gameskill == sk_easy ? 
+  if ((gameskill == sk_none && demo_version < 203) ||
```

Comment:
Why `demo_version < 203`? Oh right, it ain't fixed in Boom: https://github.com/Doom-Utils/historic-ports/blob/3ae2d25740b9d4ea896f0fde0c73f6b112135ce5/p_mobj.c#L865-L870
If you record a demo with `-skill 0` it gets out of sync in Choco and vanilla DOS exe, so it's a little weird. Maybe we should enable it just for `demo_compatibilty`?